.changeset/flat-glasses-complain.md

```diff
@@ -0,0 +1,6 @@
+---
+'hive': minor
+---
```

Comment:
This is a major change, since it will require the user to manually upgrade the database.

Reply:
Yes, they will need to do it based on their setup (ideally, they shouldn't use the community file... and use a PG instance in Cloud service, or one that was provided by their IT)

Reply:
Yeah but the changeset refers to the docker-compose file, so it kinda is a breaking change in terms of that.
Maybe we can also provide simple instructions for upgrading the local existing database (also relevant for our existing dev environments locally)

Reply:
rephrased, thanks